docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     restart: always
     build: .
     ports:
-      - 5000:5000
+      - 5050:5000
```

Comment:
Please don't alter this, or use an interpolated variable here, nginx prod expects this port to be used in the current config.  You can do things as you need in the working tree.